hydra-access-controls/app/models/concerns/hydra/admin_policy_behavior.rb

```diff
@@ -22,8 +22,9 @@ def to_solr(solr_doc=Hash.new)
     end
 
     def merged_policies
-      default_permissions.each_with_object({}) do |policy, h|
-        args = policy.to_hash
+      # Workaround for https://github.com/projecthydra/active_fedora/issues/775
```

Comment:
Ah, I think I understand. Once #775 is closed this #uniq piece doesn't have to be there.